src/randoop/sequence/ForwardGenerator.java

```diff
@@ -579,18 +573,21 @@ private InputsAndSuccessFlag selectInputs(Operation statement) {
 
       // If we were unlucky and selected a null value as the receiver
       // for a method call, return with failure.
-      if (i == 0
-          && (statement instanceof MethodCall)
-          && (!((MethodCall) statement).isStatic())
-          && chosenSeq.getCreatingStatement(randomVariable) instanceof NonreceiverTerm)
+      if (i == 0 && 
+          operation.isMessage() && 
```

Comment:
I personally find it easier to read a multi-line expression when the operators are at the beginning of the line.  (I agree the code doesn't line up quite as nicely with the operators at the beginning of the line, though.)
More importantly, the rest of the file puts the operators at the beginning of the line, and we should try to keep them consistent since inconsistent code is harder to read than either style.  Thanks!
(This applies a few other places in the diff, too.)